src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -47,24 +48,19 @@ public void storeMessage(Message message) {
   }
 
   /**
-   * Gets messages posted by a specific user.
+   * Loads in messages from the query into a list
    *
-   * @return a list of messages posted by the user, or empty list if user has never posted a
+   * @return a list of messages posted by the user(s), or empty list if user has never posted a
    *     message. List is sorted by time descending.
```

Comment:
This is a small thing, but the sort order of the List now depends on the PreparedQuery parameter, right? So you might move this comment to the other functions instead of putting it here?

Reply:
Sure thing! Would the description from the comment in my PR suffice?